spec/validators/future_date_validator_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+class Validatable
+  include ActiveModel::Validations
+  attr_accessor :date
+  validates :date, future_date: true
+end
+
+RSpec.describe FutureDateValidator do
+  subject { Validatable.new }
+
+  context "when date is nil" do
+    it "is invalid" do
+      expect(subject).to be_invalid
+    end
+
+    it "adds error message about nil" do
+      subject.validate
+      expect(subject.errors.messages[:date].first).to eql("date is nil")
+    end
+  end
+
+  context "when date is in the past" do
+    before { subject.date = Faker::Time.backward(2) }
+
+    it "is invalid" do
+      expect(subject).to be_invalid
+    end
+
+    it "adds error message about the past date" do
+      subject.validate
+      expect(subject.errors.messages[:date].first).
```

Comment:
Actually this assertion code (with indentation) has exactly 80 chars, so you don't need to split it into two lines.